tasks/options/axe-webdriver.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+	firefox: {
```

Comment:
Why only firefox? We do chrome and firefox for the performance testing id like to do the same here if possible